tutorials/issue1235.ipynb

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
<p>I don't like the names. To me, "to_lower" sounds like a name of the projection, not the projected quantity.</p><p>Also, could you please add a comment on the use of trace? I.e. explain that we need a face quantity in the higher-dimensional subdomain?</p><p>Btw., could you please check whether we use "higher dimensional" or "higher-dimensional" elsewhere? I suspect it's the latter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Regarding hyphen/no hyphen in "higher-dimensional": The majority of the occurrences throughout the codebase includes the hyphen. There are however a significant amount of non-hyphenated ones (1/3). For "lower-dimensional" vs. "lower dimensional": About a sixth of the occurrences do not include hyphen.